packages/codegen-ui-react/lib/amplify-ui-renderers/form.ts

```diff
@@ -49,6 +48,7 @@ export default class FormRenderer extends ReactComponentRenderer<BaseComponentPr
     );
 
     this.importCollection.addImport('@aws-amplify/ui-react', this.component.componentType);
```

Comment:
useTypeCastFields needs to be added to @aws-amplify/ui-react import

Reply:
just pushed an update to include that in the import, thanks